toolkits/springer/packages/springer-product-header/README.md

```diff
@@ -0,0 +1,45 @@
+# Springer Product Header
+
+The title, print cover, imprint logo and themed background colour. The title is an important signifier to recognise relevancy. Cover images are an additional visual aid to help users familiarise.
+
+Use the Product header for all pages that feature a Journal, Book or aggregations of content, such as Book series. The cover image comes from DDS. Colour is taken from the cover image automatically. Both the cover and title should link to the homepage of the title.
+
+## Usage
+
+```html
+<div class="c-product-header">
+    <!-- #not mandatory -->
+    <div class="c-product-header__theme" style="background-image: url('https://media.springernature.com/dominant-colour/springer-static/cover/journal/123.jpg')"></div>
```

Comment:
Might be worth mentioning that we get the dominant color from spindoctor in the `background-image`? Basically its not something generated from this component